design/sketch.webidl

```diff
@@ -428,14 +428,46 @@ interface WebGPURenderPipeline {
 // COMMAND RECORDING (Command buffer and all relevant structures)
 // ****************************************************************************
 
-enum WebGPULoadOp {
+/// Common interface for render and compute pass encoders.
+interface WebGPUCommandEncoder {
+    WebGPUCommandBuffer end_pass();
+    // Allowed in both compute and render passes
+    void setPushConstants(WebGPUShaderStageFlags stage,
```

Comment:
Did we agree on this yet?

Reply:
nope, this piece is just moved here, not added :)
but yes, I'm totally fine with removing it until we reach any sort of consensus on push constants